src/cascadia/TerminalApp/lib/TerminalAppLib.vcxproj

```diff
@@ -255,6 +260,15 @@
       <AdditionalIncludeDirectories>..;$(OpenConsoleDir)\dep\jsoncpp\json;%(AdditionalIncludeDirectories);</AdditionalIncludeDirectories>
       <!-- Manually disable unreachable code warning, because jconcpp has a ton of that. -->
       <DisableSpecificWarnings>4702;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+      <RuntimeTypeInfo Condition="'$(Configuration)|$(Platform)'=='AuditMode|ARM64'">true</RuntimeTypeInfo>

```

Comment:
We wouldn't add a copy of this for every permutation. We'd add it once. Presuming that we're OK adding it. See my other comment where I referenced #4023

Reply:
If your're just worried about performance impact, then I think it would be negligible. Computation time only affects the `dynamic_cast` itself, and storage space overhead for `type_info` structs only affect polymorphic classes, which there would be just 7 in the project. And I believe that link-time-optimization should strip out unused data like type names.

Reply:
I think the performance impact is likely acceptable given that this particular component is mostly UI driven and most users don't operate on the microsecond scale, which is all the impact I would expect out of this. I just want to have the discussion with my team about making such a configuration change verbally when they get back from the holiday before committing it. And I'm mildly amused that both this PR and #4023 need it at the same time.

The only thing I'd ask you to do here specifically is remove the Condition and make it be one line instead of 8-12 of them.

Reply:
Woah this is crazy that we both need it. Obviously I'm not privy to any _verbal_ discussions of the perf impacts, but I think with the points you've outlined this is fine to add.